openmc/tallies.py

```diff
@@ -1625,6 +1625,10 @@ def _align_tally_data(self, other):
         other_mean = copy.deepcopy(other.mean)
         other_std_dev = copy.deepcopy(other.std_dev)
 
+        # Initialize list of tile and repeat factors
```

Comment:
perhaps elaborate the comment with "... tile and repeat factors for filters, scores and nuclides"